src/App.js

```diff
@@ -1,16 +1,39 @@
 import React from 'react';
 import './App.css';
+// import ChatEntry from './components/ChatEntry';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+
 
 const App = () => {
+  const [chatEntries, setChatEntries] = useState(chatMessages)
+  const [likeCount, setLikeCount] = useState(0);
+  
+  const heartLikeCount = (id) => {
+    let likeCount = 0;
+    const newChatEntries = chatEntries.map((ChatEntry) => {
+      const newChatEntry = {...ChatEntry};
```

Comment:
Start with a lowercase letter for instances of a component. Starting with an upper case should be for overall component definition.